### PR TITLE
Sdlttf 02

### PR DESCRIPTION
**Pull request by andrewbird**
.

### CODE REVIEW

File: src/plugin/sdl/sdl.c [hunk 1]
```diff
@@ -225,14 +225,16 @@ static int sdl_load_font(const char *name)
   FcPatternGetString(match, FC_FILE, 0, (FcChar8 **)&pth);
   FcPatternGetInteger(match, FC_INDEX, 0, &idx);
 
-  // Fontconfig guesses if not an exact match, which is not what we want
-  if (strcasecmp(name, foundname) != 0) {
+  // Fontconfig guesses if not an exact match, which might be what we want
+  if (strncasecmp(name, foundname, min(strlen(name), strlen(foundname))) != 0) {
```
Comment (stsp) [inline src/plugin/sdl/sdl.c:229]: Maybe just foundname rather than min?

Comment (stsp) [reply src/plugin/sdl/sdl.c:229]: Or can it really be larger?
I.e. if we put only "Source Code" will
it expand to "Source Code Pro" itself?

Comment (andrewbird) [reply src/plugin/sdl/sdl.c:229]: I don't know what the matching rules are, and I'm not able to test again at the moment.

**Comment by stsp** [comment]
OK, I pushed with min() removed.
Please just get it back if your testing
will reveal its needed.
Thanks!

**Pull request closed without merging**